jest.config.js

```diff
@@ -24,14 +24,15 @@ module.exports = {
             branches: 0
         }
     },
-    roots: ['<rootDir>/testing/jest'],
+    roots: ['<rootDir>/testing/jest', '<rootDir>/js/renovation'],
     moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx'],
     preset: 'ts-jest',
     setupFiles: [
         path.join(path.resolve('.'), './testing/jest/setup-enzyme.ts'),
     ],
     testMatch: [
-        '<rootDir>/testing/jest/**/*.tests.[jt]s?(x)'
+        '<rootDir>/testing/jest/**/*.tests.[jt]s?(x)',
+        '<rootDir>/js/renovation/**/*.test.[jt]s?(x)'
```

Comment:
Maybe more strict
'<rootDir>/js/renovation/**/__tests__/*.test.[jt]s?(x)'

Additions plus. All tests that are not in __tests__ folder are ignored

Reply:
I'll do it ;)